superduperdb/base/config.py

```diff
@@ -52,8 +52,8 @@ class Cluster(JSONable):
     distributed: bool = False
     deserializers: t.List[str] = Factory(list)
     serializers: t.List[str] = Factory(list)
-    dask_scheduler: str = 'tcp://localhost:8786'
-    local: bool = True
+    dask_scheduler: str = 'tcp://scheduler:8786'
```

Comment:
Does this work without problems? We had this issue with MongoDB.

Reply:
It does not Linux, not for Mac. 

I 'll revert everything to localhost.